source/Introduction.rst

```diff
@@ -1,17 +1,24 @@
-.. _InstallationGuide:
-
 Introduction
 ============
 
-TODO:
+Space ROS is an open-source software framework, derived from `ROS 2 <https://docs.ros.org/en/humble/index.html>`_ that will be hardened to be compatible with the demands of safety-critical space robotics applications.
+Conforming to the ROS 2 Application Programming Interface (API), Space ROS is designed to be platform independent, portable and project independent.
```

Comment:
```suggestion
Conforming to the ROS 2 Application Programming Interface (API), Space ROS is designed to be platform independent, portable and, project independent.
```

Consider an Oxford comma [(pleasantly innapropriate justification)](https://www.verbicidemagazine.com/2011/09/20/strippers-jfk-and-stalin-illustrate-why-you-should-use-the-serial-comma/)

Reply:
Added.